tests/e2e/support/objects/app-files/resource/index.ts

```diff
@@ -46,6 +47,14 @@ export class Resource {
     return downloads
   }
 
+  async downloadFilesBatchAction(args: Omit<downloadResourcesArgs, 'page'>): Promise<Download[]> {
```

Comment:
can't we just use the download action for that?
Then we have to extend it to support `via: quickaction | sidebar`.